Scripts/Python/Neighborhood.py

```diff
@@ -81,3 +81,54 @@ def OnFirstUpdate(self):
             
     def OnNotify(self,state,id,events):
         pass
+
+    def _AddClothingFromAgeSDL(self, clothingList, *sdls):
+        ageSDL = PtGetAgeSDL()
+        for i in sdls:
+            if not ageSDL[i][0]:
+                PtDebugPrint(f"Neighborhood._AddClothingFromAgeSDL(): You don't have the SDL '{i}' so no '{clothingList}' for you", level=kWarningLevel)
+                return
+
+        if isinstance(clothingList, str):
+            clothingList = [clothingList]
+
+        avatar = PtGetLocalAvatar()
+        if avatar.avatar.getAvatarClothingGroup() == kFemaleClothingGroup:
+            clothingPrefix = "FReward"
+        else:
+            clothingPrefix = "MReward"
+
+        entireClothingList = avatar.avatar.getEntireClothingList(0)
+        for i in filter(None, clothingList):
+            desiredName = "_".join((clothingPrefix, i))
```

Comment:
```suggestion
            desiredName = f"{clothingPrefix}_{i}"
```
I could be wrong, but I think generally f-strings will be preferable and perform better than `.join` or `+` in Python.